### ISSUE TITLE
cosmetic: rename to benchopt in lowercase?

### ISSUE THREAD

**Issue by mathurinm** [benchopt/benchOpt#249]
We have BenchOpt, Benchopt and benchopt appearing in the doc, the GH organization is named benchopt while the main repo is benchOpt.

My 2 cents is that all lower case is easier. It is far from being critical.

**Comment by agramfort** [comment]
I thought the same too but do we want to rename the repo at this point ?

**Comment by tomMoral** [comment]
We can as the repo name is case insensitive: https://github.com/benchopt/BeNcHoPt

**Comment by tomMoral** [comment]
I agree that this was not a super wise choice to get this extra upper case
